src/PicsumProvider.php

```diff
@@ -36,6 +36,29 @@ public static function picsumUrl($width = 640, $height = 480, $id = null, $rando
         return $baseUrl . $url . $queryString;
     }
 
+    public static function picsumUrlStatic($width = 640, $height = 480, $gray = false, $blur = null)
```

Comment:
Can you rename this method to picsumRandomUrl or something like this?
IMO the "Static" word confuse users. If we call this method 2 times we get different images.

Reply:
Yes you are right. I think "picsumStaticRandomUrl" makes it less confusing. It's mentioned as "Static Random" in picsums document.